PurchasesCoreSwift/Purchasing/ReceiptFetcher.swift

```diff
@@ -0,0 +1,95 @@
+//
+//  ReceiptFetcher.swift
+//  Purchases
+//
+//  Created by Javier de Martín Gil on 8/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+// TODO: Make internal after migration to Swift is complete
+@objc(RCReceiptFetcher) public class ReceiptFetcher: NSObject {
+    private let requestFetcher: StoreKitRequestFetcher
+    private let receiptBundle: Bundle
+
+    @objc public convenience init(requestFetcher: StoreKitRequestFetcher) {
+        self.init(requestFetcher: requestFetcher, bundle: .main)
+    }
+
+    init(requestFetcher: StoreKitRequestFetcher, bundle: Bundle) {
```

Comment:
this init is only used for tests, to pass in a mock bundle

Reply:
this is `internal`, but it felt like a good idea to keep it close to the public one. Also, the public one will be internal after the migration.